charts/aws-ebs-csi-driver/templates/controller.yaml

```diff
@@ -279,6 +280,7 @@ spec:
             - --kube-api-qps=20
             - --kube-api-burst=100
             - --worker-threads=100
+            - --retry-interval-max=5m
```

Comment:
Why 5 minutes for attacher and not 30 like the others? Also, isn't this the default?

Reply:
First this is a 2-way door. In my opinion, customer misconfigurations for volume provisioning, Snapshots, and resize/modify are common enough for volume/snapshot creation and modification, which is why I think we should bump the values for those in this PR today. We can always bump attacher retry-interval-max in a followup PR. 

My worry with attachment is that it has a higher likelihood of failures ocurring for the 10+ minutes to get past the 5 min backoff. An example that comes to mind is karpenter instance detach/termination race delay for Windows taking up to 15 minutes. We would want to retry as close as possible to end of that instance termination to prevent an even longer delay. With today's 5m backoff, we'd in the worst case take 20m to attach. With a higher backoff retry-interval-max that could stretch even longer.

On the other hand there is the common attachment misconfig of attaching too many volumes. But I think in the attach path it is better to err on the side of retrying more often than the other RPCs, even with this common misconfig.